.github/workflows/pr.yaml

```
@@ -82,3 +104,58 @@ jobs:
         env:
           HAPPO_API_KEY: ${{ secrets.HAPPO_API_KEY }}
           HAPPO_API_SECRET: ${{ secrets.HAPPO_API_SECRET }}
+
+  deploy-docs:
+    if: ${{ github.event.pull_request.head.ref != 'changeset-release/master' }}
+    name: Deploy Picasso docs
+    runs-on: ubuntu-latest
+    needs: [ build-docker-image ]
+
+    steps:
+      - name: Trigger doc deployment job
+        uses: toptal/jenkins-job-trigger-action@1.0.0
+        env:
+          JENKINS_JOB_NAME: picasso-docs
+          BRANCH: ${{ github.event.pull_request.head.ref }}
+          PR_ID: ${{ github.event.pull_request.number }}
+        with:
+          jenkins_url: https://jenkins-build.toptal.net/
+          jenkins_user: toptal-jenkins
+          jenkins_token: ${{ secrets.TOPTAL_JENKINS_BUILD_TOKEN }}
+          proxy: ${{ env.HTTP_PROXY }}
+          job_name: ${{ env.JENKINS_JOB_NAME }}
```

```suggestion
          job_name: picasso-docs
```
maybe the variable is not that needed?

---

@OleksandrNechai you are probably right. Env vars are useful for such things as `github.event.pull_request.head.ref` which is pretty hard to understand, so var name helps you. But for strings - probably not needed 👍 refactored